src/data_one.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
This is similar to how Firebase is set up now. The `Items` are structured differently, with different fields, but the overall concept is the same.